tests/integration/test_metrics.py

```diff
@@ -1241,35 +1200,40 @@ def test_generic_metric_extraction(mini_sentry, relay):
             }
         ],
     }
+    config["transactionMetrics"] = {"version": 1}
```

Comment:
in order to emit the metrics in pop relays, we have to sample the envelope

Reply:
Shouldn't we set this to `"version": 3` then?

Reply:
makes sense yeah! I just copied from other tests